docs/services/ihatemoney.md

```diff
@@ -0,0 +1,46 @@
+# ihatemoney
+
+[ihatemoney](https://github.com/spiral-project/ihatemoney) is a self-hosted shared budget manager, that this playbook can install, powered by the [ansible-role-ihatemoney](https://github.com/IUCCA/ansible-role-ihatemoney) Ansible role.
+
+
+## Dependencies
+
+This service requires the following other services:
+- a [Postgres](postgres.md) database
+- a [Traefik](traefik.md) reverse-proxy server
+
+
+## Configuration
+
+To enable this service, add the following configuration to your `vars.yml` file and re-run the [installation](../installing.md) process:
+
+```yaml
+########################################################################
+#                                                                      #
+# ihatemoney                                                              #
+#                                                                      #
+########################################################################
+
+ihatemoney_enabled: true
+
+# To enable the Admin dashboard, first generate a hashed password with:
+# docker run -it --rm --entrypoint ihatemoney ihatemoney/ihatemoney generate_password_hash
```

Comment:
Maybe we can also suggest a `docker exec` command to run on the server once the service is up and running?

.. for folks that would prefer to do that.

Reply:
I have changed this to docker exec